geode-assembly/src/acceptanceTest/java/org/apache/geode/ssl/CertificateRotationTest.java

```diff
@@ -339,16 +348,16 @@ private void startCluster() throws IOException, GeneralSecurityException {
         .sanIpAddress(InetAddress.getByName("127.0.0.1"))
         .generate();
 
-    clusterKeyStore = temporaryFolder.newFile("cluster-keystore.jks");
-    writeCertsToKeyStore(clusterKeyStore.toPath(), clusterCert);
+    clusterKeyStore = createFile(rootFolder.resolve("cluster-keystore.jks"));
+    writeCertsToKeyStore(clusterKeyStore, clusterCert);
 
-    clusterTrustStore = temporaryFolder.newFile("cluster-truststore.jks");
-    writeCertsToTrustStore(clusterTrustStore.toPath(), caCert);
+    clusterTrustStore = createFile(rootFolder.resolve("cluster-truststore.jks"));
+    writeCertsToTrustStore(clusterTrustStore, caCert);
 
-    clusterSecurityProperties = temporaryFolder.newFile("cluster-security.properties");
+    clusterSecurityProperties = createFile(rootFolder.resolve("cluster-security.properties"));
     Properties properties = CertStores.propertiesWith("all", "any", "any",
         clusterTrustStore, dummyStorePass, clusterKeyStore, dummyStorePass, true, true);
-    properties.store(new FileOutputStream(clusterSecurityProperties), "");
+    properties.store(new FileOutputStream(clusterSecurityProperties.toFile()), "");
```

Comment:
Close file stream? Why does this one pass on Windows?

Reply:
That's written under a `TemporaryFolder`, which tries to delete its contents, but shrugs if it fails.

Reply:
Maybe my theory was wrong, and that unclosed file stream wasn't the problem after all.